pages/data-design/branches/workspace-notifications.mdx

```diff
@@ -28,18 +9,37 @@ Configure notifications to be sent to a Slack channel
   - Ready for review
   - Reviewer requested
   - Approved
-  - Request changes
+  - Changes requested
   - Merged
   - Closed
   - Example:
-    - Solvi approved the `onboarding-funnel` branch 🎉
+    - Solvi approved the onboarding-funnel branch 🎉
 - Comments:
   - New comment added
   - Examples:
-    - Solvi commented on the `Account Created` event on the `onboarding-funnel` branch: \_“Should we include the `role` property?”
-    - Stefania commented on the `authentication config` property on the `main` branch: _“Should we have `auto login` as a property value here when we auto login the customer from the landing page?”_
+    - Solvi commented on the Account Created event on the onboarding-funnel branch: _“Should we include the role property?”
+    - Stefania commented on the authentication config property on the main branch: “Should we have auto login as a property value here when we auto login the customer from the landing page?”
 - Collaborators:
   - Collaborator added
   - Collaborator removed
   - Example:
-    - Stefania added Solvi as a collaborator on the `onboarding-funnel` branch
+    - Stefania added Solvi as a collaborator on the onboarding-funnel branch
+
+## Enabling Slack notifications
```

Comment:
Also don't see this structural change represented in the build (moving this section to below the first one)